### PR TITLE
Update Intel VA-API drivers

### PR DESCRIPTION
**Pull request by st3r4g**
Tested on a YT video with `mpv` on `x86_64-musl`

### CODE REVIEW

**Comment by ericonr** [comment]
Oh nice, I was about to look into this one. Will test it out in a bit. Have you heard of any potential regressions?

**Comment by ericonr** [comment]
Do you want to pull in a `libva` and `libva-intel-driver` update into this?

**Comment by st3r4g** [comment]
> Have you heard of any potential regressions?

Nope

> Do you want to pull in a `libva` and `libva-intel-driver` update into this?

Good idea, the new `libva` version is listed as dependency

File: srcpkgs/intel-media-driver/template [hunk 1]
```diff
@@ -13,7 +13,7 @@ maintainer="Stefano Ragni <st3r4g@protonmail.com>"
 license="MIT, BSD-3-Clause"
 homepage="https://github.com/intel/media-driver"
 distfiles="https://github.com/intel/media-driver/archive/intel-media-${version}.tar.gz"
-checksum=1cdd40517d9fee51e3760beea23d2a19c2d5fcb1d6a9ed2bc0af7318d0d3100f
+checksum=491411d6ccdf812e4fe30b870f0623431fe9f14db16fc6b987578769925c8be0
```
Comment (ericonr) [inline srcpkgs/intel-media-driver/template:16]: I got `1c333061479064637ad4ce2a6ea07fd0d4936d4a1da5d462b88a9f5b907ace9a` here :/

**Comment by daniel-eys** [comment]
* Let's not forget about `libva-glx` and have it back in sync with `libva`.
* We could probably update `libva-utils` as well, just to have the whole vaapi stack updated.
* `libva-intel-driver`:
  * I can confirm this working on an Intel Ivybridge with glibc.
  * The `CFLAGS="-fcommon"` shouldn't be necessary anymore, looks like upstream fixed gcc10 issues [not verified though, just read the change/gitlog].
  * The python2 sed in `pre_configure()` looks obsolete.

File: srcpkgs/intel-media-driver/template (hunk 1 above)
Comment (st3r4g) [reply srcpkgs/intel-media-driver/template:16]: redownloaded the tarball, I get the same checksum:
```
$ sha256sum hostdir/sources/intel-media-driver-20.3.0/intel-media-20.3.0.tar.gz
491411d6ccdf812e4fe30b870f0623431fe9f14db16fc6b987578769925c8be0  hostdir/sources/intel-media-driver-20.3.0/intel-media-20.3.0.tar.gz
```

Comment (ericonr) [reply srcpkgs/intel-media-driver/template:16]: I will try again in a bit, then.

**Comment by st3r4g** [comment]
>       * The `CFLAGS="-fcommon"` shouldn't be necessary anymore, looks like upstream fixed gcc10 issues [not verified though, just read the change/gitlog].

Let's wait for gcc10 maybe

>       * The python2 sed in `pre_configure()` looks obsolete.

didn't understand the comment for that...

**Comment by travankor** [comment]
Any blockers for this PR or is it good to go?

**Comment by st3r4g** [comment]
I think it's good to go, no problems on my system.

> Let's wait for gcc10 maybe

Here I meant 'wait before removing that flag from the template'

**Comment by travankor** [comment]
I haven't noticed any issues for this PR in brief testing (tested media driver on BDW glibc). 

Also what is libva-glx? Is that the backend that AMD uses? The package names says GLX even though it pulls in wayland dependencies which seems weird.

**Comment by st3r4g** [comment]
> Also what is libva-glx? The package name says GLX even though it pulls in wayland dependencies which seems weird.

No idea. I wondered about that too but didn't have time to investigate

**Comment by travankor** [comment]
I think it can be removed, or at least merged into the main libva template.

**Comment by ericonr** [comment]
We can probably make `libva-glx` into a subpackage of `libva`, assuming it can be built like that.

**Comment by ericonr** [comment]
Or not, see 89904906f82462e516d00b122db73b9bcd644487

**Comment by ericonr** [comment]
Unless we can build OpenGL applications without headers from Mesa, I don't think it will be possible to remove `libva-glx`.

I built `libva-intel-driver` with `CFLAGS=-fno-common` and it worked fine, so I also removed it.

Thanks!

**Comment by st3r4g** [comment]
> Unless we can build OpenGL applications without headers from Mesa, I don't think it will be possible to remove `libva-glx`.

wait a second, we have libglvnd now. Maybe libglvnd-devel is enough for libva

**Pull request closed without merging**